### PR TITLE
fix/1860 add possibility to use env variable with uwsgi

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Related to #1860 

- [x] add possibility to use env variable with uwsgi
- [x] ~~add information about elasticsearch variable~~
- [x] add some documentation about docker image usage

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enoug
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [ ] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

**Comment by inkhey** [comment]
added dev command to help getting list of parameters

**Comment by PhilippeAccorsi** [comment]
Do not merge this branch.
@buxx and @inkhey work on it

**Comment by PhilippeAccorsi** [comment]
@lebouquetin can you make a review and merge if its ok for you?

File: tools_docker/Debian_Uwsgi/entrypoint.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# Create file with all docker variable about TRACIM parameter
+printenv |grep TRACIM > /var/tracim/data/tracim_env_variable
```
Comment (lebouquetin) [inline tools_docker/Debian_Uwsgi/entrypoint.sh]: tracim_env_variable => tracim_env_variables

Comment (PhilippeAccorsi) [reply tools_docker/Debian_Uwsgi/entrypoint.sh]: Done.

**Pull request merged**